src/components/MarkdownProvider/index.tsx

```diff
@@ -6,15 +6,57 @@
  */
 
 import React from 'react';
+import { createGlobalStyle } from 'styled-components';
 import { MDXProvider } from '@mdx-js/react';
-import { MDXRenderer } from 'gatsby-plugin-mdx';
 
-import './markdown.css';
+import { CodeExample } from './components/CodeExample';
+import { PackageDescription } from './components/PackageDescription';
+import { PropSheets } from './components/PropSheets';
+import { H2, H3, H4, H5, H6, P } from './components/typography';
 
-export const MarkdownProvider: React.FC<{ children: string }> = ({ children }) => (
-  <div className="markdown-body">
-    <MDXProvider components={{}}>
-      <MDXRenderer>{children}</MDXRenderer>
+const GlobalStyle = createGlobalStyle`
+  .anchor {
+    visibility: hidden;
+    margin-left: -20px;
+    padding-right: 4px;
+    line-height: 1;
```

Comment:
Since `color` below uses `theme`, would the `padding` and `margin` use them values too? Is `getLineHeight` suitable here?

Reply:
Forgot to theme all of this global styling. Let me add that in...